src/main/java/org/jabref/gui/Main.css

```diff
@@ -1,69 +1,243 @@
-/*
- * The base css file defining the style that is valid for every pane and dialog.
- */
-
 .root {
+
     /*
-     * A strong blue to indicate something neutral.
-     */
-    -fx-neutral: #2980b9;
+    The theme color and some derived colors from it are used for icons, tab-headers, marking of selected inputs and
+    hover colors for the main menu. It generally defines the look of JabRef. The highlighting colors below should
+    work nicely with this base color
+    */
+
+    /*-jr-theme: #2467ab;*/
+
+    /* This theme is the original JabRef dark blue color */
+    -jr-theme: #50618F;
+    /* The base gray. Most gray-tones in the application are derived from this color. */
+    -jr-base: #f0f0f0;
+
+    -jr-white: #ffffff;
+    -jr-gray-00: #f5f5f5;
+    -jr-gray-0: #f0f0f0;
+    -jr-gray-1: #e6e6e6;
+    -jr-gray-2: #808080;
+    -jr-gray-3: #404040;
+    -jr-black: #000;
+
+    /* Highlights */
+    -jr-blue: #0abde3;
+    -jr-light-blue: #48dbfb;
+    -jr-purple: #f368e0;
+    -jr-light-purple: #ff9ff3;
+    -jr-green: #10ac84;
+    -jr-light-green: #1dd1a1;
+    -jr-red: #ee5253;
+    -jr-light-red: #ff6b6b;
+    -jr-yellow: #feca57;
+    -jr-orange: #ff9f43;
+
+    /* Some blueish greys */
+    -jr-blue-gray-1: #c8d6e5;
+    -jr-blue-gray-2: #8395a7;
+    -jr-blue-gray-3: #576574;
+    -jr-blue-gray-4: #222f3e;
+
+
+    /* maintable and group view cells*/
+    -jr-hover-cell: derive(-jr-yellow, 50%);
+    -jr-selected-cell: derive(-jr-yellow, 5%);
+
+    /* Background-color of the toolbar */
+    -jr-toolbar: derive(-jr-base, 80%);
+
+    /* All icons on toolbars or next to inputs */
+    -jr-icon: -jr-theme;
+
+    /* Colors for messages and errors */
+    -jr-info: -jr-light-green;
+    -jr-warn: -jr-orange;
+    -jr-error: -jr-light-red;
+
+    /* Background-color for the small group view indicator for the number of hits */
+    -jr-group-hits-bg: -jr-gray-0;
+
+    /* Specific color for general tooltips */
+    -jr-tooltip-bg: -jr-info;
+    -jr-tooltip-fg: -jr-black;
+
+    /* Consistent size for headers of tab-pane and side-panels*/
```

Comment:
Move this down to the end, since right now it appears in the middle of color definitions?